library/src/interactor_impl.cxx

```diff
@@ -476,4 +493,25 @@ void interactor_impl::InitializeAnimation(vtkImporter* importer)
   this->Internals->AnimationManager.Initialize(
     &this->Internals->Options, this, &this->Internals->Window, importer);
 }
+
+//----------------------------------------------------------------------------
+void interactor_impl::selectBestInteractor()
+{
+  vtkRenderWindow* renWin = this->Internals->Window.GetRenderWindow();
+  vtkF3DRenderer* renderer = vtkF3DRenderer::SafeDownCast(renWin->GetRenderers()->GetFirstRenderer());
+  double bounds[6], lengths[3];
+  renderer->ComputeVisiblePropBounds(bounds);
+  vtkBoundingBox box(bounds);
+  box.GetLengths(lengths);
+  bool is2D = (lengths[0] == 0 || lengths[1] == 0 || lengths[2] == 0);
```

Comment:
Wdyt @Meakk ? I dont really like the idea of this in the libf3d directly, but it can make sense from a user POV.

Reply:
It is a simple way to deduce the best interactor style according the data dimension. As you can see, it is put in a dedicated method and called by F3DStarter, so lib user has choice to use it or not.

Reply:
yes, but instead of choosing, having a way to know if the data is 2d and react accordingly may be better.

Reply:
Indeed, I would prefer having a `loader.getSceneBoundingBox()` and a `interactor.setMode()` and put the logic in the application instead.

Reply:
I think an option to control it is better for that case, instead of a method.